.prettierignore

```diff
@@ -4,3 +4,4 @@ tmp/**
 **/types/**
 storybook-static/**
 packages/coverage/**
+packages/design-system-tokens/src/**
```

Comment:
Could this just be the `themes` and `tokens` folders? It would be nice to keep the code files in `lib` formatted in a standard way.